src/util.js

```diff
@@ -0,0 +1,45 @@
+import { readFile, writeFile, stat } from 'fs'
+import Bluebird from 'bluebird'
+
+function dequote (input) {
+  input = input.trim()
+  const singleQuote = input.startsWith('\'') && input.endsWith('\'')
+  const doubleQuote = input.startsWith('"') && input.endsWith('"')
+  if (singleQuote || doubleQuote) {
+    return input.slice(1).slice(0, -1)
+  }
+  return input
+}
+
+function readStreamAsync (stream) {
+  return new Bluebird((resolve) => {
+    let input = ''
+    stream.setEncoding('utf-8')
+    stream.on('data', x => { input += x })
+    stream.once('end', () => resolve(dequote(input)))
+    stream.resume && stream.resume()
+  })
+}
```

Comment:
I would probably move this to CLI since that's the only place it's used.  Doesn't make a lot of sense in other contexts